client/src/actions/load-group-messages.js

```diff
@@ -0,0 +1,35 @@
+import actionType from '../actionTypes';
+import apiHandler from '../components/helpers/api-handler';
+
```

Comment:
#### Critical 
Please add a `jsdoc` tag to all your methods/functions. This gives other devs a quick context concerning the code